src/main/java/org/mskcc/cmo/metadb/service/impl/ValidRequestCheckerImpl.java

```diff
@@ -101,6 +101,31 @@ public String getFilteredValidRequestJson(String requestJson) throws IOException
         return null;
     }
 
+    /**
+     * Evaluates request metadata and returns a boolean based on whether the request data
+     * passes all sanity checks.
+     * @throws JsonProcessingException or JsonMappingException
+     */
+    @Override
+    public Boolean isValidRequestMetadataJson(String requestJson)
```

Comment:
follow the same sanity checker as `getFilteredValidRequestJson` without the samples sanity checking